test/Microsoft.AspNetCore.Server.KestrelTests/FrameResponseStreamTests.cs

```diff
@@ -61,6 +61,9 @@ public void BeginReadThrows()
             var stream = new FrameResponseStream(new MockFrameControl());
             Assert.Throws<NotSupportedException>(() => stream.BeginRead(new byte[1], 0, 1, null, null));
         }
+#elif NETCOREAPP2_0
```

Comment:
Please add a comment `// Safe guard to fail compilation when TFMs have been changed in the .csproj but not in code

Easy to forget about this in the future and remove it.